2.99.4 does not compile on Mac

In file included from /Users/alexnaverniuk/xmrig/src/backend/cpu/CpuBackend.cpp:43:
/Users/alexnaverniuk/xmrig/src/3rdparty/rapidjson/document.h:1680:22: error: call to constructor of
      'rapidjson::GenericValue<rapidjson::UTF8<char>, rapidjson::MemoryPoolAllocator<rapidjson::CrtAllocator> >' is
      ambiguous
        GenericValue v(value);
                     ^ ~~~~~

--- a/src/backend/cpu/CpuBackend.cpp
+++ b/src/backend/cpu/CpuBackend.cpp
@@ -143,9 +143,9 @@ xmrig::CpuBackend::~CpuBackend()
 }
 
 
-std::pair<size_t, size_t> xmrig::CpuBackend::hugePages() const
+std::pair<unsigned, unsigned> xmrig::CpuBackend::hugePages() const
 {
-    std::pair<size_t, size_t> pages(0, 0);
+    std::pair<unsigned, unsigned> pages(0, 0);
 
 #   ifdef XMRIG_ALGO_RANDOMX
     if (d_ptr->algo.family() == Algorithm::RANDOM_X) {

--- a/src/backend/cpu/CpuBackend.h
+++ b/src/backend/cpu/CpuBackend.h
@@ -46,7 +46,7 @@ public:
     CpuBackend(Controller *controller);
     ~CpuBackend() override;
 
-    std::pair<size_t, size_t> hugePages() const;
+    std::pair<unsigned, unsigned> hugePages() const;
 
 protected:
     bool isEnabled() const override;

